diff -up libsigrok-0.2.2/configure.ac.ftdi1 libsigrok-0.2.2/configure.ac
--- libsigrok-0.2.2/configure.ac.ftdi1	2014-06-10 16:05:50.042485899 +0200
+++ libsigrok-0.2.2/configure.ac	2014-06-10 16:06:20.938025413 +0200
@@ -284,10 +284,10 @@ esac
 AM_CONDITIONAL(NEED_USB, test "x$have_libusb1_0" != xno)
 
 # libftdi is only needed for some hardware drivers. Disable them if not found.
-PKG_CHECK_MODULES([libftdi], [libftdi >= 0.16],
-	[CFLAGS="$CFLAGS $libftdi_CFLAGS";
-	LIBS="$LIBS $libftdi_LIBS";
-	SR_PKGLIBS="$SR_PKGLIBS libftdi"],
+PKG_CHECK_MODULES([libftdi1], [libftdi1 >= 1.1],
+	[CFLAGS="$CFLAGS $libftdi1_CFLAGS";
+	LIBS="$LIBS $libftdi1_LIBS";
+	SR_PKGLIBS="$SR_PKGLIBS libftdi1"],
 	[HW_ASIX_SIGMA="no"; HW_CHRONOVU_LA8="no"; HW_IKALOGIC_SCANAPLUS="no"])
 
 # libudev is only needed for some hardware drivers. Disable them if not found.
